source/uwp/AdaptiveCardsObjectModel/AdaptiveCardsObjectModel.vcxproj

```diff
@@ -0,0 +1,330 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just want to be sure (as this setting might be inherited) -- is this building with the `/w4 /wx` compiler switches?

Reply:
Yup:
![image](https://user-images.githubusercontent.com/9091833/127373153-e0ebbfb4-826f-4669-bb60-b223a77d1e48.png)

I started by just making a copy of the renderer project so the details here are mostly copied from there.